### PR TITLE
Remove callback

### PR DESCRIPTION
**Pull request by kozak**
Unwrap callback.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "phantomjs-prebuilt": "^2.1.16",
     "psc-package": "^3.0.1",
     "pulp": "15.0.0",
-    "puppeteer": "^5.4.1"
+    "puppeteer": "^5.4.1",
+    "purescript": "^0.13.8"
```
Comment (kozak) [inline package.json:25]: Doesn't build on newer versions it seems.

Comment (zyla) [reply package.json:25]: NB: this is not needed, we use a postinstall script for this. https://github.com/restaumatic/purescript-specular/pull/73/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R7

**Review by zyla — COMMENTED** [review]
Looks good; let's remove the Callback module completely though.

File: src/Specular/Ref.purs [hunk 2]
```diff
@@ -3,24 +3,27 @@ module Specular.Ref
...
   , pureFocusRef
   , previewRef
-  , updateRef
   , readRef
 
   , constRef
 
   , wrapViewWidget
-
+  , Callback
```
Comment (zyla) [inline src/Specular/Ref.purs:18]: I'd remove even the type alias, and just use `a -> Effect Unit`.

Comment (zyla) [reply src/Specular/Ref.purs]: Wow, that was _fast_ @kozak  :)

Comment (kozak) [reply src/Specular/Ref.purs]: Done

**Review by zyla — APPROVED** [review]
LGTM

Please add a changelog entry though - this is a breaking change after all.

**Pull request merged**